invenio_cli/helpers/cli_config.py

```diff
@@ -82,6 +87,20 @@ def python_package_manager(self) -> PythonPackageManager:
                 "Could not determine the Python package manager, please configure it."
             )
 
+    @property
+    def javascript_package_manager(self) -> JavascriptPackageManager:
+        """Get javascript packages manager."""
+        manager_name = self.config[CLIConfig.CLI_SECTION].get(
+            "javascript_package_manager"
+        )
+        if manager_name == NPM.name:
+            return NPM()
+        elif manager_name == PNPM.name:
+            return PNPM()
+
+        # fallback to NPM?
```

Comment:
`npm` would be the backwards-compatible default option

Reply:
yes, `npm` should be the default javascript package manager